src/patternfly/components/Label/label--variants.hbs

```diff
@@ -0,0 +1,93 @@
+{{#if label--status}}
+  {{> label
+      label--id=(concat label--variants--id '-status-default')
+      label-text--value=(concat label--variants--title)}}
+  {{> label
+      label--IsRemovable=true
+      label--id=(concat label--variants--id '-status-removable')
+      label-text--value=(concat label--variants--title ' removable')}}
+  {{> label
+      label-content--IsLink=true
+      label--id=(concat label--variants--id '-status-link')
+      label-text--value=(concat label--variants--title ' link')}}
+  {{> label
+      label-content--IsButton=true
+      label--id=(concat label--variants--id '-status-button')
+      label-text--value=(concat label--variants--title ' button')}}
+  {{> label
+      label--IsRemovable=true
+      label-content--IsLink=true
+      label--id=(concat label--variants--id '-status-link-removable')
+      label-text--value=(concat label--variants--title ' link removable')}}
+  {{> label
+      label--IsRemovable=true
+      label-content--IsButton=true
+      label--id=(concat label--variants--id '-status-button-removable')
+      label-text--value=(concat label--variants--title ' button removable')}}
+  {{> label
+      label--id=(concat label--variants--id '-status-truncated')
+      label-text--value=(concat label--variants--title ', max-width truncation customization')}}
+  {{> label
+      label--IsRemovable=true
+      label--id=(concat label--variants--id '-status-truncated-removable')
+      label-text--value=(concat label--variants--title ' removable, max-width truncation customization')}}
+{{else}}
+  {{> label
+      label--id=(concat label--variants--id '-default')
+      label-text--value=label--variants--title}}
+
+  {{> label
+      label--id=(concat label--variants--id '-icon')
+      label-icon--value="info-circle"
```

Comment:
Just a nit: since we're showing status labels that would use this icon, we might want to avoid using a "statusy" icon outside a status context. WdYT?